tests/layer_tests/tensorflow_tests/test_tf_MatrixInverse.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestMatrixInverse(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'input:0' in inputs_info
+        input_shape = inputs_info['input:0']
+        inputs_data = {}
+        inputs_data['input:0'] = np.random.rand(*input_shape).astype(np.float32)  # Example random input
```

Comment:
for test you need to provide inversible matrix, i.e. det A !=0. Please think how to generate such matrix

Reply:
I add a if condition to check if the generated matix has the correct rank, and the its determinant shouldn't be 0.